aiida_sssp_workflow/workflows/workfunctions.py

```diff
@@ -0,0 +1,7 @@
+"""helper workfunction"""
```

Comment:
Something like this is already implemented in `aiida-optimize`, right?

Reply:
Yes, but it is in the `aiida_optimiza/tests, so I'd better keep it in this library. 

well, since it will not be used in practice, I remove it to to the `tests` directory.